pkg/autohealing/controller/controller.go

```diff
@@ -392,6 +403,7 @@ func (c *Controller) Start(ctx context.Context) {
 			}
 
 			wg.Wait()
+			c.provider.UpdateHealthStatus(masterUnhealthyNodes, workerUnhealthyNodes)
```

Comment:
It would be better to update cluster status before the repair action is triggered, the repair action is a long time operation, I know it's limited by the current goroutine repair design, do you think it's acceptable for now?

Reply:
I can see your point. My understanding is, given the repairing is a time-consuming operation, so it should be acceptable to do the status update here. I will do more testing about this. Thanks.